OpenRA.Mods.Common/ServerTraits/LobbyCommands.cs

```diff
@@ -95,6 +95,8 @@ public bool InterpretCommand(S server, Connection conn, Session.Client client, s
 				{ "option", Option(server, conn, client) },
 				{ "assignteams", AssignTeams(server, conn, client) },
 				{ "kick", Kick(server, conn, client) },
+				{ "promote", Promote(server, conn, client) },
```

Comment:
IMO these names could be made a bit more consistent.  How about something like "make_admin" / `MakeAdmin` and "make_spectator" / `MakeSpectator`?